include/SDL_render.h

```diff
@@ -877,6 +877,19 @@ extern DECLSPEC int SDLCALL SDL_RenderFillRects(SDL_Renderer * renderer,
                                                 const SDL_Rect * rects,
                                                 int count);
 
+/**
+ *  \brief Fill some number of triangles on the current rendering target with the drawing color.
+ *
+ *  \param renderer The renderer which should fill multiple triangles.
+ *  \param points   A pointer to an array of destination points representing a triangle strip.
```

Comment:
Is a triangle strip the only option people need? It might be out of the scope of this specific PR but I always imagined something like GL_TRIANGLES + a list of vertices + an index array. If the initial API only supports triangle strips instead of that, maybe we should make sure the APIs are forward-looking in terms of their naming or parameters.

Triangle strips definitely seem less useful to me than triangles + indices when rendering textured geometry, at least.

Reply:
I've looked briefly at the older implementation from Tom and it does use triangles + indices rather than a triangle strip, fwiw.